data_model/src/permission.rs

```diff
@@ -67,11 +38,11 @@ mod model {
         Getters,
     )]
     #[ffi_type]
-    #[display(fmt = "PERMISSION `{id}` = `{payload}`")]
+    #[display(fmt = "PERMISSION `{name}` = `{payload}`")]
     #[getset(get = "pub")]
     pub struct Permission {
         /// Refers to a type defined in [`crate::executor::ExecutorDataModel`].
-        pub id: PermissionId,
+        pub name: Name,
```

Comment:
changed to `name` because `id` implies that it's `Identifiable` which is not the case and has been a source of confusion many times